internal/handler/company_handler.go

```
@@ -0,0 +1,55 @@
+	err = h.createUsecase.CreateCompany(req)
+	if err != nil {
+		w.WriteHeader(http.StatusBadRequest)
+		json.NewEncoder(w).Encode(res.ErrorResponse{
+			Message: "error",
+			Error:   err.Error(),
+		})
+		return
+	}
+
+	w.WriteHeader(http.StatusCreated)
+	json.NewEncoder(w).Encode(res.SuccessResponse{
```

golangci-lint (1.64.8)

Error return value of `(*encoding/json.Encoder).Encode` is not checked

(errcheck)